core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -178,6 +186,9 @@ public LockResource lockExclusive(StateLockOptions lockOptions)
       // Activate the interrupt cycle before entering the lock because it might wait in the queue.
       activateInterruptCycle();
       // Force the lock.
+      LOG.debug("Thread-%d forcing the lock with %d waiters/holders: {%s}",
+          Thread.currentThread().getId(), mSharedWaitersAndHolders.size(), mSharedWaitersAndHolders
+              .stream().map((th) -> Long.toString(th.getId())).collect(Collectors.joining(",")));
```

Comment:
I think that we might want to wrap this in a if (Log.isDebugEnabled()) because grabbing iterating, and joining into a String can be pretty expensive?

Reply:
I made it INFO now. This is OK to be seen imo.